tests/test_resources/test_clusters/conftest.py

```diff
@@ -411,24 +411,30 @@ def local_docker_cluster_telemetry_public_key(request, detached=True):
         client.images.prune()
 
 
-@pytest.fixture(scope="session")
-def local_test_account_cluster_public_key(request, test_account):
-    container_name = "rh-slim-test-acct"
-    detached = request.config.getoption("--detached")
-    den_auth = "den_auth" in request.keywords
-
-    keypath = str(
-        Path(rh.configs.get("default_keypair", DEFAULT_KEYPAIR_KEYPATH)).expanduser()
-    )
-    local_ssh_port = BASE_LOCAL_SSH_PORT + 3
+@pytest.fixture(scope="function")
+def local_test_account_cluster_public_key(request, test_account, detached=True):
+    """
+    This fixture is not parameterized for every test; it is a separate cluster started with a test account
+    (username: kitchen_tester) in order to test sharing resources with other users.
+    """
+    with test_account as test_account_dict:
+        image_name = "keypair"
+        container_name = "rh-slim-test-acct"
+        dir_name = "public-key-auth"
+        detached = request.config.getoption("--detached")
+        den_auth = "den_auth" in request.keywords
+        keypath = str(
+            Path(
+                rh.configs.get("default_keypair", DEFAULT_KEYPAIR_KEYPATH)
+            ).expanduser()
+        )
+        local_ssh_port = BASE_LOCAL_SSH_PORT + 4
```

Comment:
At some point we'll have to do something about the hardcoded offsets in the fixtures..

Reply:
I think docker network is the durable answer